### PR TITLE
Feature/dsnd 2789 fix

### PR DESCRIPTION
**Pull request by SALMANCH007**

### CODE REVIEW

**Review by YaaseenR — APPROVED** [review]

File: src/test/java/uk/gov/companieshouse/pscdataapi/PscDataApiApplicationTests.java [hunk 1]
```diff
@@ -2,10 +2,15 @@
 
 import org.junit.jupiter.api.Test;
 import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import uk.gov.companieshouse.api.api.CompanyExemptionsApiService;
 
 @SpringBootTest
 class PscDataApiApplicationTests {
 
+//    @MockBean
```
Comment (JAndrewCH) [inline src/test/java/uk/gov/companieshouse/pscdataapi/PscDataApiApplicationTests.java:11]: remove commented out mock

File: src/test/java/uk/gov/companieshouse/pscdataapi/api/ResourceChangedApiServiceAspectFeatureFlagDisabledITest.java [hunk 2]
```diff
@@ -53,6 +54,9 @@ class ResourceChangedApiServiceAspectFeatureFlagDisabledITest {
     @MockBean
     private ChsKafkaApiService mapper;
 
+//    @MockBean
```
Comment (JAndrewCH) [inline src/test/java/uk/gov/companieshouse/pscdataapi/api/ResourceChangedApiServiceAspectFeatureFlagDisabledITest.java:57]: can remove

File: src/test/java/uk/gov/companieshouse/pscdataapi/api/ResourceChangedApiServiceAspectFeatureFlagEnabledITest.java [hunk 3]
```diff
@@ -49,6 +50,9 @@ class ResourceChangedApiServiceAspectFeatureFlagEnabledITest {
     @Mock
     private RequestExecutor requestExecutor;
 
+//    @MockBean
```
Comment (JAndrewCH) [inline src/test/java/uk/gov/companieshouse/pscdataapi/api/ResourceChangedApiServiceAspectFeatureFlagEnabledITest.java:53]: can remove

**Review by JAndrewCH — CHANGES_REQUESTED** [review]

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Review by JAndrewCH — APPROVED** [review]

**Pull request merged**